src/Service.Tests/GraphQLBuilder/Helpers/GraphQLTestHelpers.cs

```diff
@@ -88,20 +88,50 @@ public static Entity GenerateEmptyEntity(SourceType sourceType = SourceType.Tabl
         }
 
         /// <summary>
-        /// Creates a stored procedure backed entity using the provided metadata.
+        /// Creates a stored procedure backed entity using the supplied metadata.
         /// </summary>
         /// <param name="graphQLTypeName">Desired GraphQL type name.</param>
         /// <param name="graphQLOperation">Query or Mutation</param>
-        /// <param name="permissionOperations">Collection of permission operations (CRUD+Execute)</param>
+        /// <param name="permissionOperations">Collection of permission operations (CRUD+Execute). Default: Execute</param>
+        /// <param name="dbObjectName">Name of object in the database. Default: foo</param>
+        /// <param name="parameters">Collection of defined stored procedure parameters</param>
+        /// <param name="keyFields">The field(s) to be used as primary keys.</param>
```

Comment:
Since stored procedure entities can never have key-fields, why do we take this as an input?

Reply:
This was an error, thank you for catching. Removed.